demo/public/index.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
+    <link rel="shortcut icon" type="image/png" href="%PUBLIC_URL%/favicon.png">
+    <title>React, GraphQL and Contentful App</title>
```

Comment:
I don't know if I should apply `App` to `Contentful` only or to the whole title. Maybe:

> React App backed by Contentful and GraphQL